### DISCUSSION TITLE
WezTerm has no icon under Wayland (shows Wayland "W" icon)

### DISCUSSION THREAD

**Discussion by ludgie** [wezterm/wezterm#5842] [Q&A]
### What Operating System(s) are you running on?

Linux Wayland, Fedora 40

### Which Wayland compositor or X11 Window manager(s) are you using?

KDE Plasma 6.1.2 with KWin 6.1.2

### WezTerm version

wezterm 20240721_002142_1654c9d1 (nightly repo from copr)

### Ask your question!

WezTerm has no icon under Wayland on the taskbar, while on Plasma kickoff it has one.
How can I fix this issue?

![image](https://github.com/user-attachments/assets/cd4b79f3-2047-4a9f-8c64-9a456bb107b7)
![image](https://github.com/user-attachments/assets/c977a0cc-29c1-4b52-8a0c-9d6dba589572)

**Comment by ludgie** [comment]
And running `wezterm start --class org.wezfurlong.wezterm --always-new-process` spawns for a moment a WezTerm icon and then it goes to the W icon.

**Reply by g0r3** [reply]
Experiencing the same issue. It appeared after upgrading from the repo version to the git version of wezterm on Arch Linux

**Comment by alexpls** [comment]
Hey! Are you using fractional scaling? If yes, this sounds like a duplicate of: https://github.com/wez/wezterm/issues/6081

**Reply by ludgie** [reply]
Hello! That's my issue. I think I already linked this discussion with the issue. I'll just close this discussion. Thanks still.
